firmware/Core/Src/rtos_tasks/rtos_tasks.c

```diff
@@ -197,26 +208,123 @@ void TASK_service_eps_watchdog(void *argument) {
 	// Important to service the watchdog near the start, though.
 	osDelay(10000);
 
-	while(1) {
+	while (1)
+	{
 		const uint8_t result = EPS_CMD_watchdog();
 
-		if (result != 0) {
+		if (result != 0)
+		{
 			LOG_message(
 				LOG_SYSTEM_EPS,
 				LOG_SEVERITY_ERROR,
 				LOG_SINK_ALL,
-				"EPS_CMD_watchdog() -> Error: %d", result
-			);
+				"EPS_CMD_watchdog() -> Error: %d", result);
 		}
-		else {
+		else
+		{
 			LOG_message(
 				LOG_SYSTEM_EPS,
-				LOG_SEVERITY_NORMAL, 
+				LOG_SEVERITY_NORMAL,
 				LOG_SINK_ALL,
-				"EPS watchdog serviced successfully." 
-			);
+				"EPS watchdog serviced successfully.");
 		}
-		
+
 		osDelay(sleep_duration_ms);
 	}
 }
+
+void TASK_receive_gps_info(void *argument)
```

Comment:
I'd recommend moving this task into a separate file (perhaps `gps_rtos_tasks.c`).

Reply:
Done